main.py

```diff
@@ -289,32 +288,35 @@ def status(message):
             data=queue.peek(blocking=True)
             data={k.decode():(v.decode() if isinstance(v, bytes) else v, u.decode()) for k,(v,u) in data.items()}
             data=json.dumps(data)
-            info=client.publish("prism/"+cfg['mqtt']['uname'],data, qos=1)
+            info=client.publish("prism/{}/data".format(cfg['mqtt']['uname']),data, qos=1)
             info.wait_for_publish()
             while info.rc!=0:
                 time.sleep(10)
-                info=client.publish("prism/"+cfg['mqtt']['uname'],data , qos=1)
+                info=client.publish("prism/{}/data".format(cfg['mqtt']['uname']),data, qos=1)
                 info.wait_for_publish()
+            for sensor in input_sensors:
+                sensor.transmitted_data(len(queue))
```

Comment:
Move these two lines after the data is deleted from the queue so that `len(queue)` shows the correct size.